dask/diagnostics/profile_visualize.py

```diff
@@ -169,16 +168,9 @@ def visualize(profilers, file_path=None, show=True, save=True, **kwargs):
     The completed bokeh plot object.
     """
     bp = import_required('bokeh.plotting', _BOKEH_MISSING_MSG)
-    import bokeh
+    from bokeh.io.state import curstate
```

Comment:
This should be `from bokeh.io import curstate`, right?

Reply:
probably, `bokeh.io.state` exists from a recent refactoring, but `bokeh.io` is the stable public interface. If the intent is to support as many older versions as possible `bokeh.io` is the way to go

Reply:
Seems like `bokeh.io` doesn't import `curstate`:

```python
In [1]: from bokeh.io import curstate
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-30e0c3d1f7ce> in <module>()
----> 1 from bokeh.io import curstate

ImportError: cannot import name 'curstate'

In [2]: import bokeh

In [3]: bokeh.__version__
Out[3]: '0.12.10'
```

Should I open an issue on the Bokeh issue tracker?

Reply:
Well it looks like I unintentionally made difficult for you, unfortunately. I couldn't imagine any use cases when a regular user should ever use `curstate` so I put it in the "dev" category, which means it's only in `bokeh.io.state` as of `0.12.10` and not in the "normal user" public API that is in `bokeh.io`. I guess you are stuck with a version check (or try except) after all. My apologies.